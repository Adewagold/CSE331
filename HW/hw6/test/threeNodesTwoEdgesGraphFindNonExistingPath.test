# Tests find path on three_nodes_two_edges_graph.

# Load threeNodesTwoEdgesGraph.tsv
LoadGraph three_nodes_two_edges_graph threeNodesTwoEdgesGraph.tsv

# Find minimum-cost path from node1 to node3
FindPath three_nodes_two_edges_graph node1 node3