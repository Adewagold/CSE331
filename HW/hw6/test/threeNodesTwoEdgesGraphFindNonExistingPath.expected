# Tests find path on three_nodes_two_edges_graph.

# Load threeNodesTwoEdgesGraph.tsv
loaded graph three_nodes_two_edges_graph

# Find minimum-cost path from node1 to node3
path from node1 to node3:
no path found